### PR TITLE
Adding tests to increase coverage

### PR DESCRIPTION
**Pull request by maliagehan**
### Checklist:

- [ ] Relevant tests (and test data) have been added or updated and passed.
- [ ] The documentation was added or updated.
- [ ] Relevant issues were updated or added.

### CODE REVIEW

File: plantcv/shift_img.py [hunk 1]
```diff
@@ -59,9 +59,8 @@ def shift_img(img, device, number, side="right", debug=None):
         adjust = iy - number
         adjusted_img = np.hstack((left, ori_img[0:, 0:adjust]))
 
-
     if len(np.shape(img)) == 2:
-        adjusted_img, channel2, channel3 = np.dsplit(adjusted_img, 3)
+        adjusted_img=adjusted_img[:,:,0]
```
Comment (nfahlgren) [inline plantcv/shift_img.py]: For PEP8 the format would be `adjusted_img = adjusted_img[:, :, 0]`

File: tests/tests.py [hunk 2]
```diff
@@ -36,6 +36,7 @@
 TEST_INTPUT_MULTI = "multi_ori_image.jpg"
 TEST_INPUT_MULTI_CONTOUR = "roi_objects.npz"
 TEST_INPUT_ClUSTER_CONTOUR = "clusters_i.npz"
+TEST_INPUT_GENOTXT="cluster_names.txt"
```
Comment (nfahlgren) [inline tests/tests.py]: PEP8 recommends spaces around operators like `=`

Comment (nfahlgren) [reply tests/tests.py]: Unless they are in function calls...

Comment (maliagehan) [reply tests/tests.py]: fixed it!

File: plantcv/shift_img.py (hunk 1 above)
Comment (maliagehan) [reply plantcv/shift_img.py]: fixed it!

**Pull request merged**